packages/core/src/components/steps-modal/steps-modal.scss

```diff
@@ -0,0 +1,9 @@
+@import '~@atomium/scss-utils/index';
+
+.hide {
+  display: none;
+}
+
+.show {
+  margin-top: var(--spacing-base);
+}
```

Comment:
(block) the class name is not declarative with it goal

Reply:
since this one is being made directly on the component I changed its name to be only step [693539c](https://github.com/juntossomosmais/atomium/pull/599/commits/693539c1609736fc4c7eaffbc565d0631bc03425)

Reply:
should we use the BEM pattern as shadow: false?

Reply:
GOOD POINT @tassioFront